### PR TITLE
Clarify that frames are compound objects.

### PR DESCRIPTION
**Pull request by kilimanjaro**
This is minor but important, because we often speak of the "sample rate", "frequency", etc of a frame, when these things are meaningful only for a fully qualified frame (on a specific set of qubits).

### CODE REVIEW

File: rfcs/analog/spec_changes.md [hunk 1]
```diff
@@ -183,8 +187,8 @@ SET-SCALE 0 "xy" 0.75
 **Capture**
 
 ```
-Capture :: CAPTURE Qubit Frame Waveform MemoryReference
-RawCapture :: RAW-CAPTURE Qubit Frame Expression MemoryReference
+Capture :: CAPTURE Qubit FrameName Waveform MemoryReference
```
Comment (kilimanjaro) [inline rfcs/analog/spec_changes.md]: We're baking in a bit extra by assuming that `CAPTURE`s happen on single-qubit frames, but I'm ok with that.

Comment (stylewarning) [reply rfcs/analog/spec_changes.md]: Why not `Frame` with the condition that it's a 1q frame, just for a clean grammar?

Comment (kilimanjaro) [reply rfcs/analog/spec_changes.md]: I'm fine with that too (tbh I wasn't sure why Steven et al restricted to 1 qubit).

Comment (kilimanjaro) [reply rfcs/analog/spec_changes.md]: Actually I don't think much is lost by just allowing capture on multi-qubit frames, at least at the level of the quilt spec. Frames have no intrinsic meaning here anyways. Actual quilt programs on our near-term hardware will ofc be using the available readout frames.

Comment (caryan) [reply rfcs/analog/spec_changes.md]: No need to keep the 1q condition. There are architectures with multiple qubits attached to the same readout resonator.

**Comment by caryan** [comment]
:ok_hand: this compound structure make a lot of sense to me. Is it worth being explicit in the spec about why we tie frames to qubits? In my mind frames are perfectly meaningful independent of qubits - I can blast microwaves at the chip without caring about qubits; however, we tie them to qubits in QUILT to enable scheduling.

File: rfcs/analog/spec_changes.md (hunk 1 above)
Comment (kilimanjaro) [reply rfcs/analog/spec_changes.md]: Just clarify @caryan, in the multi-qubit `CAPTURE` would it be appropriate to have a waveform for each qubit, or simply one for the whole resonator? Likewise would there be a single resulting IQ value? Along similar lines, what would multi-qubit `RAW-CAPTURE` look like?

Comment (caryan) [reply rfcs/analog/spec_changes.md]: Hmmmm... typically it's a single raw trace that gets integrated to a single IQ value that gives some linear combination of all the pauli Z-terms (for 2 qubits ZI, IZ and ZZ).  But in this case one  could also have multiple waveforms to get more information from the time series data.

**Pull request merged**